docs/docs/zh/guide/howto/datasource-setting.md

```diff
@@ -4,7 +4,7 @@
 
 我们这里以 MySQL 为例来说明如何配置外部数据库：
 
-> 如果使用 MySQL 需要手动下载 [mysql-connector-java 驱动][mysql] (8.0.16) 并移动到 DolphinScheduler 的每个模块的 libs 目录下，其中包括 `api-server/libs` 和 `alert-server/libs` 和 `master-server/libs` 和 `worker-server/libs`。
+> 如果使用 MySQL，需要手动下载 [mysql-connector-java 驱动][mysql] (8.0.16) 并解压后移动到 DolphinScheduler 的每个模块的 libs 目录下，其中包括 `api-server/libs`、`alert-server/libs`、`master-server/libs`、`worker-server/libs`、`tools/libs`。
```

Comment:
Just for double-check, since DS removes the dependency of `DAO` module in worker, do we still need to add `mysql-connector-java` for worker? https://github.com/apache/dolphinscheduler/pull/13242 cc @ruanwenjun

Reply:
No, we don't need to add mysql driver for worker, unless you need to use MySQL in task plugin.